app/models/person.rb

```diff
@@ -267,6 +267,8 @@ class Person
     {name: "person_broker_agency_staff_role_id_search"}
   )
 
+  index({ "verification_types.validation_status" => 1 })
```

Comment:
do we really need this indexing ? (as this is only for report, i guess we don't need this)